core/src/main/java/dk/alexandra/fresco/lib/math/integer/binary/RightShift.java

```diff
@@ -49,12 +46,9 @@
    * @param bitLength An upper bound for the bitLength of the input.
    * @param input The input.
    * @param calculateRemainders true to also calculate remainder. If false remainders in result will
-   * be null.
+   *        be null.
    */
-  public RightShift(
-      int bitLength,
-      Computation<SInt> input,
-      boolean calculateRemainders) {
+  public RightShift(int bitLength, Computation<SInt> input, boolean calculateRemainders) {
```

Comment:
I have a hard time tracking the changes here - is it just realignment?

Reply:
Yea :)

Reply:
Agree - try to be careful with the realignments in the future. It resides in the commit "Initial work on binary version. Does not compile, but got a dummy test to run. So far so good." which is not accurate then.